src/JuliusSweetland.OptiKey/UI/ValueConverters/OverrideHasValue.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace JuliusSweetland.OptiKey.UI.ValueConverters
+{
+    public class OverrideHasValue : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return value != null && !value.Equals("") && value.ToString() != "";
```

Comment:
Could this be replaced with !string.IsNullOrEmpty?

Reply:
Yes, that seems to work. I've replaced it with
`return value != null && !string.IsNullOrEmpty(value.ToString());`
is that correct?